### PR TITLE
Changed user location dot color to #3070B3

### PR DESCRIPTION
**Pull request by Marius1501**
I changed the color of the user location dot to a TUM-color (#3070B3). The corresponding issue is #119.

### CODE REVIEW

File: webclient/src/views/view/view-view.scss [hunk 1]
```diff
@@ -458,6 +458,15 @@
...
     }
 
+    /* --- User location dot --- */
+    .mapboxgl-user-location-dot {
+        background-color: #3070B3;
+    }
+
+    .mapboxgl-user-location-dot::before {
+        background-color: #3070B3;
+    }
```
Comment (octycs) [inline webclient/src/views/view/view-view.scss:468]:
```suggestion
    .mapboxgl-user-location-dot, .mapboxgl-user-location-dot::before {
        background-color: #3070B3;
    }
```

**Review by octycs — APPROVED** [review]
Thanks for your PR! I hope I don't scare you away with these comments at once: You can merge the two definitions into one by using a comma. I think it's better this way. It would also be also possible to write:
```scss
.mapboxgl-user-location-dot {
    background-color: #3070B3;
    &::before {
        background-color: #3070B3;
    }
}
```
(it's one of the cool features of Sass, but because it''s the same color I think it's better to use the comma version I think)

For variables that change with the theme we have the colors all defined in `webclient/src/variables.scss`. But the map doesn't change with the theme at the moment, so in my opinion it's not necessary to create a variable there.

**Comment by octycs** [comment]
Nice :+1: I've just seen that there are now two blank lines above and below the `background-color: #3070B3;` :sweat_smile:

**Comment by Marius1501** [comment]
Oh, should I delete them? 😄

**Comment by octycs** [comment]
Yes, would be great. We usually format them like
```scss
selector {  // No space here
    prop: value;
    another-prop: value;
                                    // maybe a space here for readability
    final-prop: value;
}  // No space here
```

**Comment by Marius1501** [comment]
Thank you! That`s my first merge! 😄

**Pull request merged**